src/templates/profile.ts

```diff
@@ -10,8 +13,104 @@ version = "${version}"
 export function empty(name: string): string {
   return `
 """
-${name} usecase
+${name} use case
+Use case description
 """
 usecase ${name} {}
 `;
 }
+export function completeProfile({
+  scope,
+  name,
+  version,
+  usecaseNames,
+}: {
+  name: string;
+  scope?: string;
+  version: string;
+  usecaseNames: string[];
+}): string {
+  const id = ProfileId.fromScopeName(scope, name);
+  const formattedName = startCase(id.id);
+
+  const error = errorModel(scope);
+
+  return [
+    `"""
+${formattedName}
+TODO: What "${formattedName}" does
+"""
+name = "${id.id}"
+version = "${version}"
+// Comlink Profile specification: https://superface.ai/docs/comlink/profile`,
+    ...usecaseNames.map(u => usecase(u, error.statement)),
+    error.namedModel,
+  ].join('\n');
+}
+
+function usecase(useCaseName: string, errorStatement: string): string {
+  const formattedName = startCase(useCaseName);
+
+  return `
+"""
+${formattedName}
+TODO: What "${formattedName}" does
+"""
+usecase ${useCaseName} unsafe { // change safety to \`safe\`, \`idempotent\` or \`unsafe\`
+  input {
+    "Foo title"
+    foo! string 
+  }
+
+  result {
+    bar string
+  }
+
+  error ${errorStatement}
+
+  example success {
+    input {
+      foo = "example"
+    }
+    result {
+      bar = "result"
+    }
+  }
+
+  example fail {
+    input {
+      foo = "error"
+    }
+    error {
+      title = "Not Found"
+      detail = "Entity not found"
+    }
+  }
```

Comment:
As discussed during the call, let's use less nerdy (meme-like) default field names

```suggestion
  input {
    """
    Field Title
    This field has a specific meaning for the use case. You can describe it here
    """
    field! string
  }

  result {
    outcomeField string
  }

  error ${errorStatement}

  example success {
    input {
      field = "example"
    }
    result {
      outcomeField = "result"
    }
  }

  example fail {
    input {
      field = "error"
    }
    error {
      title = "Not Found"
      detail = "Entity not found"
    }
  }
```